similarity_api/src/application.py

```diff
@@ -1,10 +1,37 @@
+import logging
+import atexit
+
 from flask import Flask
 from db.db import db
+from sim.compute import cache_scores as cache
+from flask_apscheduler import APScheduler
+
+scheduler = APScheduler()
 
 def create_app():
     app = Flask(__name__, static_folder='static')
+
     app.config.from_pyfile("settings.py")
+
     db.init_app(app)
+
     from routes.create import create_routes
     create_routes(app)
+
+    scheduler.init_app(app)
+    scheduler.start()
+    # scheduler.add_job(func=cache_scores, id='cache', trigger='interval', seconds=10, timezone='EST')
```

Comment:
Can this line be removed

Reply:
I thought it would be helpful for other developers to also have a way to quickly create short-interval jobs for debugging, but I can remove it. cee88711e27b46deb7e81d0072309dd3b8e1a027